tests/handler/handler_invoke_api.cpp

```diff
@@ -30,18 +30,23 @@ using accessor_t =
                        cl::sycl::access::target::global_buffer>;
 
 struct single_task_functor {
-  single_task_functor(accessor_t acc) : acc(acc) {}
+  single_task_functor(accessor_t acc, size_t range) : acc(acc), range(range) {}
```

Comment:
I believe the accessor should be able to access its own range, unless maybe we're using ranged accessors?

Reply:
The passed accessor has access to the full buffer range (256), but we only want to write `defaultRange` (128) to test whether an implementation writes more than it should (granted, highly unlikely in the single task variant, as it simply does a for-loop).